packages/flutter_tools/lib/src/commands/analyze.dart

```diff
@@ -9,28 +9,47 @@ import '../runner/flutter_command.dart';
 import 'analyze_continuously.dart';
 import 'analyze_once.dart';
 
-bool isDartFile(FileSystemEntity entry) => entry is File && entry.path.endsWith('.dart');
-
-typedef bool FileFilter(FileSystemEntity entity);
-
 class AnalyzeCommand extends FlutterCommand {
-  AnalyzeCommand({ bool verboseHelp: false, this.workingDirectory }) {
-    argParser.addFlag('flutter-repo', help: 'Include all the examples and tests from the Flutter repository.', defaultsTo: false);
-    argParser.addFlag('current-package', help: 'Include the lib/main.dart file from the current directory, if any.', defaultsTo: true);
-    argParser.addFlag('dartdocs', help: 'List every public member that is lacking documentation (only works with --flutter-repo and without --watch).', defaultsTo: false, hide: !verboseHelp);
-    argParser.addFlag('watch', help: 'Run analysis continuously, watching the filesystem for changes.', negatable: false);
-    argParser.addFlag('preview-dart-2', defaultsTo: true, help: 'Preview Dart 2.0 functionality.');
-    argParser.addOption('write', valueHelp: 'file', help: 'Also output the results to a file. This is useful with --watch if you want a file to always contain the latest results.');
-    argParser.addOption('dart-sdk', valueHelp: 'path-to-sdk', help: 'The path to the Dart SDK.', hide: !verboseHelp);
+  AnalyzeCommand({bool verboseHelp: false, this.workingDirectory}) {
+    argParser.addFlag('flutter-repo',
+        negatable: false,
+        help: 'Include all the examples and tests from the Flutter repository.',
+        defaultsTo: false,
+        hide: !verboseHelp);
+    argParser.addFlag('current-package',
+        help: 'Analyze the current project, if applicable.', defaultsTo: true);
+    argParser.addFlag('dartdocs',
+        negatable: false,
+        help:
+            'List every public member that is lacking documentation (only works with --flutter-repo).',
+        hide: !verboseHelp);
+    argParser.addFlag('watch',
+        help: 'Run analysis continuously, watching the filesystem for changes.',
+        negatable: false);
+    argParser.addFlag('preview-dart-2',
+        defaultsTo: true, help: 'Preview Dart 2.0 functionality.');
+    argParser.addOption('write',
+        valueHelp: 'file',
+        help:
+            'Also output the results to a file. This is useful with --watch if you want a file to always contain the latest results.');
```

Comment:
It looks like you're going for `dartfmt` style in this file, in which case let's wrap these long strings:

```dart
       help:
           'Also output the results to a file. This is useful with --watch if '
           'you want a file to always contain the latest results.');
```

Reply:
done